### PR TITLE
task2

### PR DESCRIPTION
**Pull request by pvordgh**

### CODE REVIEW

File: task2.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+ 3. Создайте массив $arr=['a', 'b', 'c', 'd'] и с его помощью выведите на экран строку 'a+b, c+d'.
+
+ 4. Создайте массив $arr с элементами 2, 5, 3, 9. Умножьте первый элемент массива на второй, а третий элемент на четвертый. Результаты сложите, присвойте переменной $result. Выведите на экран значение этой переменной.
+
+ 5. Заполните массив $arr числами от 1 до 5. Не объявляйте массив, а просто заполните его присваиванием $arr[] = новое значение.*/
+
+
+echo "ЗД 1 <br>";
+$arr=['a', 'b', 'c'];
```
Comment (ckomarov) [inline task2.php]: code style (check PSR-12)
need spaces before and after =

File: task2.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+var_dump($arr);
```
Comment (ckomarov) [inline task2.php]: use var_dump only for debuggig purpose, don't push it

File: task2.php [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+echo "<br>";
+
+echo "ЗД 2 <br>";
+echo "первый элемент: $arr[0], второй элемнт: $arr[1], третий элемент $arr[2] <br>";
+
+echo "ЗД 3 <br>";
+$arr=['a', 'b', 'c', 'd'];
```
Comment (ckomarov) [inline task2.php]: code style

File: task2.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+echo $arr[0].'+'.$arr[1].', '.$arr[2].'+'.$arr[3].'<br>';
```
Comment (ckomarov) [inline task2.php]: code style (spaces around '.')

File: task2.php [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+echo "ЗД 4 <br>";
+$arr=[2, 5, 3, 9];
```
Comment (ckomarov) [inline task2.php]: code style

File: task2.php [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+$result = $arr[0] * $arr[1] + $arr[2] * $arr[3];
+echo $result;
+echo "<br>";
+
+echo "ЗД 5 <br>";
+echo "вариант 1 <br>";
+$array[] = 1;
```
Comment (ckomarov) [inline task2.php]: use a loop to push elements to array

File: task2.php [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+$array[] = 2;
+$array[] = 3;
+$array[] = 4;
+$array[] = 5;
+var_dump($array);
+echo "<br>";
+echo "вариант 2 <br>";
+var_dump(range(1, 5));
+?>
```
Comment (ckomarov) [inline task2.php]: don't use this

**Review by ckomarov — CHANGES_REQUESTED** [review]

**Review by ckomorov — APPROVED** [review]
done.

**Pull request merged**